plugins/woocommerce-admin/client/products/fields/attribute-field/add-attribute-modal.tsx

```diff
@@ -284,9 +287,12 @@ export const AddAttributeModal: React.FC< AddAttributeModalProps > = ( {
 										'Add another attribute',
 										'woocommerce'
 									) }
-									onClick={ () =>
-										addAnother( values, setValue )
-									}
+									onClick={ () => {
+										recordEvent(
+											'product_add_attributes_modal_add_another_attribute_button_click'
```

Comment:
I like this suggestion... it is less wordy than the suggestion I made, and therefore easier to read. I think we should update the Tracks event names in this PR to follow this tweak, rather than opening up a follow-up PR (because, ideally we don't change Tracks event names once they are in place).

> My thinking is that generally we should go from least specific to most specific with the action verb at the end

That's the guideline. :-)